### PR TITLE
admin documentation

### LINKED ISSUE #60: As user, I want to know how the admin works
A small user manual should be written to manage Objects and to manage Objecttypes.

At minimum it should explain how to configure both component so it works in practise. AUthoirzations and configuration should probably cover alot.

- [x] Creating an Objecttype (in Objecttypes Admin)
- [x] Make objecttype available for objects (in Objects Admin)
- [x] Create object (in Objects Admin)

### PR DESCRIPTION
**Pull request by annashamray**
fixes #60

### CODE REVIEW

File: docs/admin/index.rst [hunk 1]
```diff
@@ -0,0 +1,18 @@
+.. _admin_index:
+
+Admin interface
+===============
+
+Both the Objects API and the Objecttypes API have the admin interface which
```
Comment (joeribekker) [inline docs/admin/index.rst:6]:
```suggestion
Both the Objects API and the Objecttypes API have an admin interface that
```

File: docs/admin/index.rst [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+allows to configure APIs and to manage objects and related object types.
```
Comment (joeribekker) [inline docs/admin/index.rst:7]:
```suggestion
allows you to configure the APIs and to manage objects and related object types.
```

File: docs/admin/index.rst [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+Below you can find tutorials how to use admin interface to manage APIs.
```
Comment (joeribekker) [inline docs/admin/index.rst:9]:
```suggestion
Below, you can find tutorials how to use the admin interface to manage the APIs.
```

File: docs/admin/object.rst [hunk 4]
```diff
@@ -0,0 +1,67 @@
+.. _admin_object:
+
+=======
+Objects
+=======
+
+Creating and updating objects are usually done using the Objects API. :ref:`api_usage`
```
Comment (joeribekker) [inline docs/admin/object.rst:7]:
```suggestion
Creating and updating objects is usually done using the Objects API. :ref:`api_usage`
```

File: docs/admin/object.rst [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+        "boomgroep": "Solitaire boom",
+        "boomhoogteactueel": 3,
+        "leeftijd": 100,
+        "meerstammig": false
+    }
+
+If you want, you can validate your JSON data against the JSON schema on `JSONschema.dev <https://jsonschema.dev>`_
+
+In the admin page of the Objects API click on the "add" button for the "Objects"
+resource.
```
Comment (joeribekker) [inline docs/admin/object.rst:39]:
```suggestion
In the Objects admin dashboard, click on "Objects" and click on the "Add" button.
```

I like how you called the admin, the "Objects admin" in the introduction. I think we should keep this term throughout the document.

File: docs/admin/objecttype.rst [hunk 6]
```diff
@@ -0,0 +1,131 @@
+.. _admin_objecttype:
+
+============
+Object types
+============
+
+Creating and changing object types are common tasks for the Objecttypes API.
+:ref:`api_usage` describes how to do it with HTTP requests. But you can also
+use the admin interface to create, publish and update object types.
```
Comment (joeribekker) [inline docs/admin/objecttype.rst:9]:
```suggestion
use the Objecttypes admin to create, publish and update object types.
```

File: docs/admin/objecttype.rst [hunk 7]
```diff
@@ -0,0 +1,131 @@
...
+          "title": "Type",
+          "description": "Typering van het beheerobject."
+        }
+      },
+      "required": ["boomhoogteactueel", "leeftijd"]
+    }
+
+
+In the admin page of the Objects API click on the "add" button for the "Object types"
+resource.
```
Comment (joeribekker) [inline docs/admin/objecttype.rst:74]:
```suggestion
In the Objecttypes admin dashboard click on "Objecttypes" and click on the "Add" button.
```

Oops Objects API :p

**Review by joeribekker — CHANGES_REQUESTED** [review]

File: docs/admin/objecttype.rst (hunk 7 above)
Comment (annashamray) [reply docs/admin/objecttype.rst]: thanks for catching it :)

**Review by joeribekker — APPROVED** [review]

**Pull request merged**